frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar.unit.spec.js

```diff
@@ -60,26 +67,46 @@ function getDataset(card) {
   );
 }
 
-function setup({ question, cachingEnabled = true } = {}) {
+async function setup({ question, cachingEnabled = true } = {}) {
+  const user = createMockUser();
+
   const settings = mockSettings({
     "enable-query-caching": cachingEnabled,
     "query-caching-min-ttl": 10000,
   });
 
+  const id = question.id();
+  nock(location.origin)
+    .get(`/api/card/${id}`)
+    .reply(200, question.card())
+    .get(`/api/revision?entity=card&id=${id}`)
+    .reply(200, [])
+    .get("/api/user")
+    .reply(200, [user])
+    .get(`/api/user/${user.id}`)
+    .reply(200, user);
+
```

Comment:
The component was throwing a ton of errors while trying to make this requests without proper mocks